gui/tools/autorepl.py

```diff
@@ -336,7 +337,7 @@ def write_results():
             if ds_full in remote_zfslist:
                 continue
             log.debug("ds = %s, remotefs = %s" % (ds, remotefs))
-            sshproc = pipeopen('%s %s %s' % (sshcmd, rzfscmd, ds_full), quiet=True)
+            sshproc = pipeopen('%s %s \"%s\"' % (sshcmd, rzfscmd, ds_full) + "\'", quiet=True)
```

Comment:
What if we didnt change rzfscmd above and just did:

`sshproc = pipeopen('%s \'%s \"%s\"\'' % (sshcmd, rzfscmd, ds_full), quiet=True)`

wouldnt it make ir more clear? or is it used anywhere else?

Reply:
In fact, it seems we are breaking the bidirectional replication down below (if even used!?) by changing rzfscmd to have a single quote in beginning ?

Reply:
The bi-directional replication isn't used. I'll escape in the `pipeopen` function.